.editorconfig

```diff
@@ -72,7 +72,7 @@ insert_final_newline = true
 charset = utf-8
 indent_style = space
 indent_size = 4
-max_line_length = 120
+max_line_length = 100
 trim_trailing_whitespace = true
 
 [{Makefile, *.mk}]
```

Comment:
Is this file needed? Also, we usually use 120 in the other repos.

Reply:
Yes, the file is required.